From 88ae95d522bfdda4c29a1116367aad8c7d948779 Mon Sep 17 00:00:00 2001
From: Jelle van der Waa <jelle@vdwaa.nl>
Date: Sat, 21 Oct 2017 12:46:49 +0200
Subject: [PATCH] Fix no decleration of `assert` with GCC 7.2

Add a missing include for cassert to fix the -fpermissive compiler
error.
---
 include/utils/Image.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/utils/Image.h b/include/utils/Image.h
index e8c82e4..dad57a2 100644
--- a/include/utils/Image.h
+++ b/include/utils/Image.h
@@ -1,6 +1,7 @@
 #pragma once
 
 // STL includes
+#include <cassert>
 #include <vector>
 #include <cstdint>
 #include <cstring>
-- 
2.14.2

